apollo-router/tests/apollo_otel_traces.rs

```diff
@@ -0,0 +1,519 @@
+//! Be aware that this test file contains some fairly flaky tests which embed a number of
```

Comment:
If the integration test runner was used rather than the test harness then all the test could be isolated.

Reply:
Actually I haven't seen any flakiness TBH - I'll remove this line.

Reply:
https://github.com/apollographql/router/pull/4982/commits/4ad43159c8ab10d011bd62acfa27b52160084534